commands/publish/index.js

```diff
@@ -256,10 +257,31 @@ class PublishCommand extends Command {
 
     return chain.then(() => {
       const count = this.packagesToPublish.length;
-      const message = this.packagesToPublish.map((pkg) => ` - ${pkg.name}@${pkg.version}`);
 
       output("Successfully published:");
-      output(message.join(os.EOL));
+
+      if (this.options.summaryFile) {
```

Comment:
I don't think this branching makes sense... The summaryFile will be an empty string in the case where you _do_ want it and want its default location.

You haven't covered this case with tests which is why this has been missed I think

Reply:
I've updated this - don't know if this is the 'right' way in this project to handle `boolean` and `string` typed cli prompts, lmk if there is a specific way

Reply:
I'm not sure myself, I think this is fine for now